### PR TITLE
Update upgrade instructions

### PR DESCRIPTION
**Pull request by mandarjog**
1. Fix some missing comments.
2. Add more verification steps.

### CODE REVIEW

File: content/en/docs/setup/upgrade/index.md [hunk 1]
```diff
@@ -60,6 +73,16 @@ For example, the following command will show all the pods using the `canary` rev
 $ kubectl get pods -n test-ns -l istio.io/rev=canary
 {{< /text >}}
 
+To verify that the new pods in the `test-ns` namespace are using the `istiod-canary` service corresponding to the `canary` revision, select one newly created pod and use the `pod_name` in the following command:
+
+{{< text bash >}}
+$ kubectl  exec -i -t -n test-ns ${pod_name} -c istio-proxy -- curl http://localhost:15000/clusters | grep xds | grep hostname
+
+xds-grpc::10.32.6.58:15012::hostname::istiod-canary.istio-system.svc
+{{< text bash >}}
```
Comment (mandarjog) [inline content/en/docs/setup/upgrade/index.md:82]:
```suggestion
{{< /text >}}
```

File: content/en/docs/setup/upgrade/index.md [hunk 2]
```diff
@@ -60,6 +73,16 @@ For example, the following command will show all the pods using the `canary` rev
 $ kubectl get pods -n test-ns -l istio.io/rev=canary
 {{< /text >}}
 
+To verify that the new pods in the `test-ns` namespace are using the `istiod-canary` service corresponding to the `canary` revision, select one newly created pod and use the `pod_name` in the following command:
+
+{{< text bash >}}
+$ kubectl  exec -i -t -n test-ns ${pod_name} -c istio-proxy -- curl http://localhost:15000/clusters | grep xds | grep hostname
```
Comment (howardjohn) [inline content/en/docs/setup/upgrade/index.md:79]:
```
$ ik pc bootstrap POD | grep discoveryAddress
                                "discoveryAddress": "istiod.istio-system.svc:15012",
```

also works

or 

```
$ ik pc endpoints `kp alpine` --cluster xds-grpc -ojson | grep hostname
                "hostname": "istiod.istio-system.svc",
```

same concept, just avoiding complicated exec commands

**Comment by mandarjog** [comment]
```./istioctl-1.6  proxy-config endpoints svc00-0v1-6bf54cd94f-gdcn2.service-graph00 --cluster xds-grpc -ojson | grep hostname```

File: content/en/docs/setup/upgrade/index.md (hunk 2 above)
Comment (mandarjog) [reply content/en/docs/setup/upgrade/index.md]: Changed to use `istioctl proxy-config` instead of kubectl.

**Pull request merged**